WORKSPACE

```diff
@@ -0,0 +1 @@
+workspace(name = "com_googlesource_code_gflags")
```

Comment:
Seems the only reason for a `WORKSPACE` file as the one you added in this PR would be for convenience so users can use `generate_workspace` to create their `WORKSPACE` file.

> Bazel provides a tool to help generate these expansive WORKSPACE files, called generate_workspace. This is not included with the binary installer, so you'll need to clone the GitHub repo to use it. We recommend using the tag corresponding to your current version of bazel, which you can check by running bazel version.
(https://bazel.build/versions/master/docs/external.html#transitive-dependencies)